### PR TITLE
fix typos using misspell

### PR DESCRIPTION
**Pull request by aspala**
This PR is part of a campaign to fix a lot of typos on github!
You can see the progress on https://github.com/fixTypos/fix_typos/

https://github.com/client9/misspell

**-what I did:**
fixed typos within comments

**-how-to-verify**
see changes : D

**-description for the changelog**
Fixed typos

**-picture of cute animal**
img_0095

### CODE REVIEW

**Comment by albers** [comment]
LGTM - Thanks!

File: Dockerfile.windows [hunk 1]
```diff
@@ -152,7 +152,7 @@
 # -----------------------------------------------------------------------------------------
 
 
-# The number of build steps below are explicitly minimised to improve performance.
+# The number of build steps below are explicitly minimized to improve performance.
```
Comment (thaJeztah) [inline Dockerfile.windows]: 🇬🇧 :smile:

File: pkg/namesgenerator/names-generator.go [hunk 2]
```diff
@@ -417,7 +417,7 @@ var (
 		// Carla Meninsky, was the game designer and programmer for Atari 2600 games Dodge 'Em and Warlords. https://en.wikipedia.org/wiki/Carla_Meninsky
 		"meninsky",
 
-		// Johanna Mestorf - German prehistoric archaeologist and first female museum director in Germany - https://en.wikipedia.org/wiki/Johanna_Mestorf
+		// Johanna Mestorf - German prehistoric archeologist and first female museum director in Germany - https://en.wikipedia.org/wiki/Johanna_Mestorf
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: I think the original was good here

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Found one change that wasn't correct

**Comment by cpuguy83** [comment]
Ping.
Can you fix the one issue and then remove the docs changes? Docs are now in github.com/docker/cli.
Let us know, or we can carry. Thanks!

**Pull request closed without merging**